src/main/java/frc/robot/subsystems/vision/VisionConstants.java

```diff
@@ -3,7 +3,14 @@
 import edu.wpi.first.math.geometry.*;
 
 public class VisionConstants {
-    public static final Transform3d[] ROBOT_TO_CAM = new Transform3d[]{new Transform3d(new Translation3d(0, 0, 0.45), new Rotation3d(0, 0, 180)), new Transform3d(new Translation3d(0, 0, 0.45), new Rotation3d(0, 0, 0))};
+    public static final Transform3d[] ROBOT_TO_CAM = new Transform3d[]{
```

Comment:
1. No need to shorten, use the full form of the word.
2. Use the word "cameras" instead of "camera" so it will be clear there are multiple in this constant.